plugins/node/opentelemetry-instrumentation-mongodb/src/mongodb.ts

```diff
@@ -0,0 +1,412 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  getSpan,
+  SpanStatusCode,
+  Span,
+  context,
+  diag,
+  SpanKind,
+} from '@opentelemetry/api';
+import type * as mongodb from 'mongodb';
+import {
+  MongodbCommandType,
+  MongoInternalCommand,
+  MongoInternalTopology,
+  WireProtocolInternal,
+  MongoDbInstrumentationConfig,
+  CursorState,
+} from './types';
+import { VERSION } from './version';
+import {
+  DatabaseAttribute,
+  GeneralAttribute,
+} from '@opentelemetry/semantic-conventions';
+import {
+  InstrumentationBase,
+  InstrumentationNodeModuleDefinition,
+  InstrumentationNodeModuleFile,
+  isWrapped,
+} from '@opentelemetry/instrumentation';
+
+const supportedVersions = ['>=3.3 <4'];
```

Comment:
Should we still support version `2.x` as previously we did. It is quite old though (older than 3 years) and what about version  >=3.0.0 <=3.2.x ?
Is this something we should really worry about or we are can drop it ?

Reply:
I commented on the PR description that IMO it seems fine and if its requested we could re-add support, WDYT ?

Reply:
sry I missed that, not a blocker